pyproject.toml

```diff
@@ -13,7 +13,7 @@ authors = [
 license = {file = "LICENSE"}
 requires-python = ">=3.6"
 dependencies = [
-    "policyengine_core",
+    "policyengine_core=3.10",
```

Comment:
__question:__ Does this in fact require 3.10? I believe we're on 3.15. Is this necessary because of Hugging Face download changes?

Reply:
Think this was a mistake- moved to >=